account/account_test.go

```diff
@@ -1079,7 +1083,7 @@ func TestWaitForTransactionReceipt(t *testing.T) {
 //
 //	none
 func TestAddDeclareTxn(t *testing.T) {
-	// https://goerli.voyager.online/tx/0x76af2faec46130ffad1ab2f615ad16b30afcf49cfbd09f655a26e545b03a21d
+	// https://sepolia.voyager.online/tx/0x76af2faec46130ffad1ab2f615ad16b30afcf49cfbd09f655a26e545b03a21d
```

Comment:
This transaction doesn't exist on the Sepolia network.

Reply:
Didn't find this line in the code. Hence, considered fixed.